scripts/jenkins/centos67/TestPullRequest_core.sh

```diff
@@ -8,7 +8,7 @@ cd $HOOT_HOME
 hoot --version --debug
 
 export HOOT_TEST_DIFF=--diff
-make -s -f Makefile.old services-test-all
+make -s -f Makefile.hoot services-test-all
 HootTest --exclude=.*ConflateAverageTest.sh \
          --exclude=.*ExactMatchInputsTest.sh \
          --exclude=.*RafahConflateTest.sh \
```

Comment:
@sputnam should we just delete this script now that we are on CentOS 7?

Reply:
Yes.  And there are likely others I should clean out as well.